syntax = "proto2";

import "node_conf.proto";

message ActivityConfig
{
    optional string         conf_file_path = 1;     // activity配置文件地址
    required NodeConfig     node_conf = 2;          // node配置
}

message ActivityInfo
{
    optional string         class_name = 1;     // 类名，与模块的类名一致
    optional ActivityConfig activiry_conf = 2;
}
